request/models.go

```diff
@@ -8,6 +8,26 @@ type Request struct {
 	URL     string
 }
 
+// Merge merges information from another request into the original request, overwriting any data
+// that is provided in toMerge.
+func (original *Request) Merge(toMerge Request) {
+	if toMerge.Body != "" {
```

Comment:
Should these empty string checks really be
```golang
if len(strings.TrimSpace(s)) == 0
```
I can see why you might think this is overkill though...

Reply:
Yeah, it's overkill. 😄